src/parse/preprocess.ts

```diff
@@ -70,7 +39,8 @@ export function preprocessTemplateRange(
     prefix = '{/*';
     suffix = '*/}';
 
-    const nextToken = code.slice(template.range.end).toString().match(/\S+/);
+    const nextToken = sliceByteRange(code, template.range.endByte).match(/\S+/);
```

Comment:
In content-tag @v4, this likely isn't needed, as both byte ranges and char ranges are emitted now

Reply:
Hmm, maybe. I'm not too familiar with the implementation details of `content-tag`. Because there wasn't a good migration guide to `v4`, I used the same utility methods from `ember-codemod-sort-invocations` and adjusted this plugin's code to make use of them.

Maybe we can look into deleting unnecessary code as a result of `content-tag@v4` in a separate task?

Reply:
Yes, once this is all removed from a dep update pr.

Content-tag has no migration guides, cause it's a low level tool, hoping folks watch development haha